metron-platform/metron-writer/src/main/java/org/apache/metron/writer/bolt/BulkMessageWriterBolt.java

```diff
@@ -74,7 +81,11 @@ public void prepare(Map stormConf, TopologyContext context, OutputCollector coll
     this.writerComponent = new BulkWriterComponent<>(collector);
     this.collector = collector;
     super.prepare(stormConf, context, collector);
-    messageGetter = MessageGetters.valueOf(messageGetterStr);
+    if (messageGetStrategyType.equals(MessageGetters.JSON_FROM_FIELD.name()) && messageGetField != null) {
```

Comment:
Now that I think about it, you might want to change the `arg` to `String` from `Object` and either:
* insist that the implementing `MessageGetter`'s provide an appropriate constructor
* convert in the enum (e.g. `arg -> BytesFromPosition(ConversionUtils.convert(arg, Integer.class)`)